[BUG] 로그백 설정 오류로 서버가 실행되지 않는 버그 해결

## 🐞 어떤 상황에서 발생한 버그인가요?

로그백 설정 오류

```
Caused by: java.lang.IllegalStateException: Logback configuration error detected: 
ERROR in c.q.l.core.rolling.DefaultTimeBasedFileNamingAndTriggeringPolicy - Filename pattern [/app/log/%d{yyyy-MM-dd}/info/info%i.log] contains an integer token converter, i.e. %i, INCOMPATIBLE with this configuration. Remove it.
ERROR in c.q.l.core.rolling.DefaultTimeBasedFileNamingAndTriggeringPolicy - Filename pattern [/app/log/%d{yyyy-MM-dd}/warn/warn%i.log] contains an integer token converter, i.e. %i, INCOMPATIBLE with this configuration. Remove it.
ERROR in c.q.l.core.rolling.DefaultTimeBasedFileNamingAndTriggeringPolicy - Filename pattern [/app/log/%d{yyyy-MM-dd}/error/error%i.log] contains an integer token converter, i.e. %i, INCOMPATIBLE with this configuration. Remove it.
        at org.springframework.boot.logging.logback.LogbackLoggingSystem.reportConfigurationErrorsIfNecessary(LogbackLoggingSystem.java:282)
        at org.springframework.boot.logging.logback.LogbackLoggingSystem.loadConfiguration(LogbackLoggingSystem.java:260)
        at org.springframework.boot.logging.AbstractLoggingSystem.initializeWithConventions(AbstractLoggingSystem.java:81)
        at org.springframework.boot.logging.AbstractLoggingSystem.initialize(AbstractLoggingSystem.java:61)
        at org.springframework.boot.logging.logback.LogbackLoggingSystem.initialize(LogbackLoggingSystem.java:193)
        at org.springframework.boot.context.logging.LoggingApplicationListener.initializeSystem(LoggingApplicationListener.java:332)
        ... 26 more
```

https://logback.qos.ch/manual/appenders.html#TimeBasedRollingPolicy

TimeBasedRollingPolicy에서는 `%d` 토큰만 지원합니다. `%i`는 사용할 수 없어요.

## 🎁 예상 결과
정상적으로 로그 출력

## ⏳ 예상 소요 시간 (예상 해결 날짜)
10분

## 🔍 참고할만한 자료(선택)

--- a/backend/src/main/resources/logger/file/error-appender.xml
+++ b/backend/src/main/resources/logger/file/error-appender.xml
@@ -16,7 +16,7 @@
         </encoder>
 
         <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-            <fileNamePattern>${LOG_FILE_PATH}/%d{yyyy-MM-dd}/error/error%i.log</fileNamePattern>
+            <fileNamePattern>${LOG_FILE_PATH}/%d{yyyy-MM-dd}/error.log</fileNamePattern>
             <totalSizeCap>5GB</totalSizeCap>
             <maxHistory>30</maxHistory>
         </rollingPolicy>

--- a/backend/src/main/resources/logger/file/info-appender.xml
+++ b/backend/src/main/resources/logger/file/info-appender.xml
@@ -16,7 +16,7 @@
         </encoder>
 
         <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-            <fileNamePattern>${LOG_FILE_PATH}/%d{yyyy-MM-dd}/info/info%i.log</fileNamePattern>
+            <fileNamePattern>${LOG_FILE_PATH}/%d{yyyy-MM-dd}/info.log</fileNamePattern>
             <totalSizeCap>5GB</totalSizeCap>
             <maxHistory>30</maxHistory>
         </rollingPolicy>

--- a/backend/src/main/resources/logger/file/warn-appender.xml
+++ b/backend/src/main/resources/logger/file/warn-appender.xml
@@ -16,7 +16,7 @@
         </encoder>
 
         <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-            <fileNamePattern>${LOG_FILE_PATH}/%d{yyyy-MM-dd}/warn/warn%i.log</fileNamePattern>
+            <fileNamePattern>${LOG_FILE_PATH}/%d{yyyy-MM-dd}/warn.log</fileNamePattern>
             <totalSizeCap>5GB</totalSizeCap>
             <maxHistory>30</maxHistory>
         </rollingPolicy>

